defaults/config.js

```diff
@@ -368,4 +368,11 @@ module.exports = {
 	// @default  false
 	//
 	debug: false,
+
+	// Enables logging raw IRC messages into each server window.
+	//
+	// @type     boolean
+	// @default  false
+	//
+	debugRaw: false,
```

Comment:
You mean you prefer an object instead of an array?
`{ connection: true, raw: true }` doesn't seem any more explicit than `["connection", "raw"]` to me, so I must not be following you correctly.